src/test/java/hudson/plugins/copyartifact/ParameterizedBuildSelectorTest.java

```diff
@@ -131,7 +132,9 @@ public void testWorkflow() throws Exception {
         ));
         
         VirtualFile vf = b.getArtifactManager().root().child("artifact.txt");
-        assertEquals("foobar", IOUtils.toString(vf.open()));
+        try(InputStream in = vf.open()) {
+            assertEquals("foobar", IOUtils.toString(in, "UTF-8"));
+        }

```

Comment:
To release the file handle on Windows.
Windows denies deleting files when their handles are held.